components/suggest/src/schema.rs

```diff
@@ -80,6 +80,25 @@ pub const SQL: &str = "
         id TEXT PRIMARY KEY,
         data BLOB NOT NULL
     ) WITHOUT ROWID;
+
+    CREATE TABLE yelp_subjects(
+        keyword TEXT PRIMARY KEY
+    ) WITHOUT ROWID;
+
+    CREATE TABLE yelp_modifiers(
+        type INTEGER NOT NULL,
+        keyword TEXT NOT NULL,
+        PRIMARY KEY (type, keyword)
+    ) WITHOUT ROWID;
+
+    CREATE TABLE yelp_location_signs(
+        keyword TEXT PRIMARY KEY,
+        need_location INTEGER NOT NULL
+    ) WITHOUT ROWID;
+
+    CREATE TABLE yelp_templates(
+        url TEXT PRIMARY KEY
+    ) WITHOUT ROWID;
```

Comment:
I agree with @linabutler that simply storing this in memory somewhere seems simpler and sufficient. In practice, there should only ever be one template I think. If there's more than one, then it's likely because we're providing two different UXes to the user, e.g., one might be a Yelp SERP URL, another might be a direct link to a business's Yelp page or something. And in that case, it's likely that each UX will need to know its specific URL template, so storing them all together in a table doesn't really seem helpful.

Reply:
Okay, I will do it!